global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is this really necessary?

Reply:
Only if we want to lock the SKD to a specific version.
I removed this for now.